### DISCUSSION TITLE
Is there a way to switch back to Semver tags intead of PEP440 tags?

### DISCUSSION THREAD

**Discussion by shadycuz** [python-poetry/poetry#6673] [General]
Poetry used to make nice tags that I'm guessing was semver compliant. Recently it was changed to make PEP440 tags, I think here https://github.com/python-poetry/poetry-core/pull/140

In my opinion, the new prerelease tags are ugly and at a quick glance can be mistaken for a normal tag. 

Before/after the change

![image](https://user-images.githubusercontent.com/17090999/193423765-26624f88-cb3d-445f-a7f1-b346878de359.png)

EDIT: Found the loser who asked for PEP440 tags to be implemented. #3752

![image](https://user-images.githubusercontent.com/17090999/193425153-24df0775-7aac-4a8d-86a4-714dbfd21453.png)

**Comment by dimbleby** [comment]
no
